workflow/defaults.py

```diff
@@ -0,0 +1,44 @@
+"""Functions to load default parameters for EMOD-3D simulations."""
+
+import importlib
+from enum import StrEnum
+from importlib import resources
+
+import yaml
+
+
+class DefaultsVersion(StrEnum):
+    """Enum of versions that can be loaded by load_defaults."""
+
+    v24_2_2_1 = "24.2.2.1"
+    v24_2_2_2 = "24.2.2.2"
+    v24_2_2_4 = "24.2.2.4"
+    develop = "develop"
+
+
+def load_defaults(version: DefaultsVersion) -> dict[str, int | float | str]:
+    """Load default parameters for EMOD3D simulation from a YAML file.
+
+    Parameters
+    ----------
+    version : str
+        Version number of the EMOD3D parameters to load. This should be in the format 'YY.M.D.V'.
+
+    Returns
+    -------
+    dict
+        A dictionary containing the default parameters loaded from the YAML file.
+        The keys are strings representing parameter names, and the values can be
+        integers, floats, or strings depending on the parameter.
+    """
+    if version == DefaultsVersion.develop:
+        defaults_package = importlib.import_module(
```

Comment:
Do these need to be packages? Could these just be files in the default_parameters directory and then have a mapping dictionary for the StrEnum to the filename and just load that? Would make this simpler imo

Reply:
A simpler approach again would perhaps be to simply declare the defaults as variables in the `defaults.py` module. There is a trade-off though: if they're in the code it reduces the readability  somewhat for non-programmers.

Aside from that importlib approach is recommended way (per the stdlib docs) to load external resources. If we do decide to stick with external files for the defaults I would prefer to use importlib rather than `__file__` or similar.